sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_responses.hpp

```diff
@@ -171,13 +171,6 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
   } // namespace Models
 
   class StartCopyFileOperation : public Azure::Core::Operation<Models::FileProperties> {
-  public:
-    Azure::ETag ETag;
-    Azure::DateTime LastModified;
-    std::string CopyId;
-    Models::CopyStatusType CopyStatus;
-    Nullable<std::string> VersionId;
```

Comment:
CL entry needed. I think in the previous release we returned `FileStartCopyResult` which had these public fields so its a breaking change worth calling out.